### PR TITLE
Various mappings in net/minecraft/world/level/levelgen

### PR DESCRIPTION
**Pull request by PiTheGuy**

### CODE REVIEW

File: data/net/minecraft/world/level/levelgen/DensityFunctions.mapping [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+		ARG 0 noiseData
+		ARG 1 xzScale
+		ARG 3 yScale
+	METHOD noise (Lnet/minecraft/core/Holder;)Lnet/minecraft/world/level/levelgen/DensityFunction;
+		ARG 0 noiseData
+	METHOD noise (Lnet/minecraft/core/Holder;D)Lnet/minecraft/world/level/levelgen/DensityFunction;
+		ARG 0 noiseData
+		ARG 1 yScale
+	METHOD noise (Lnet/minecraft/core/Holder;DD)Lnet/minecraft/world/level/levelgen/DensityFunction;
+		ARG 0 noIseData
```
Comment (alcatrazEscapee) [inline data/net/minecraft/world/level/levelgen/DensityFunctions.mapping]:
```suggestion
		ARG 0 noiseData
```

File: data/net/minecraft/world/level/levelgen/structure/BuiltinStructureSets.mapping [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/world/level/levelgen/structure/BuiltinStructureSets
+	METHOD register (Ljava/lang/String;)Lnet/minecraft/resources/ResourceKey;
+		ARG 0 location
```
Comment (alcatrazEscapee) [inline data/net/minecraft/world/level/levelgen/structure/BuiltinStructureSets.mapping]:
```suggestion
		ARG 0 name
```
This is more analogous to the existing `register` methods which generally are using `name` or `key` (both are used interchangeably in different locations).

File: data/net/minecraft/world/level/levelgen/structure/BuiltinStructures.mapping [hunk 3]
```diff
@@ -0,0 +1,3 @@
+CLASS net/minecraft/world/level/levelgen/structure/BuiltinStructures
+	METHOD createKey (Ljava/lang/String;)Lnet/minecraft/resources/ResourceKey;
+		ARG 0 location
```
Comment (alcatrazEscapee) [inline data/net/minecraft/world/level/levelgen/structure/BuiltinStructures.mapping]:
```suggestion
		ARG 0 name
```

**Review by alcatrazEscapee — CHANGES_REQUESTED** [review]

File: data/net/minecraft/world/level/levelgen/DensityFunctions.mapping (hunk 1 above)
Comment (PiTheGuy) [reply data/net/minecraft/world/level/levelgen/DensityFunctions.mapping]: Oops, I didn't even notice that.

**Comment by PiTheGuy** [comment]
Everything should be fixed

**Review by alcatrazEscapee — APPROVED** [review]

**Pull request merged**